docs/cosmology/dev.rst

```diff
@@ -107,33 +106,32 @@ the definition of :class:`~astropy.cosmology.FLRW`.
 
         def __init__(self, H0, Om0, Ode0, Tcmb0=0.0*u.K, Neff=3.04, m_nu=0.0*u.eV,
                      Ob0=None, *, name=None, meta=None):
-            cls = self.__class__
```

Comment:
Note: do update the parameter definitions, since these now have `fvalidate`!

Reply:
Done!